2/contrib/s2i/run

```diff
@@ -230,12 +230,19 @@ if [[ "${CONTAINER_MEMORY_IN_BYTES}" -lt $((2**40)) ]]; then
   fi
 fi
 
+FATAL_ERROR_OPTION=""
+
+if [[ "${FATAL_ERROR_LOG}" == "true" ]]; then
+    echo "Fatal error log is true. We need to set the Error File to reside in the PV."
+    FATAL_ERROR_OPTION="-XX:ErrorFile=/var/lib/jenkins/logs"
```

Comment:
the PR discussion implied the path would be configured by the env var.  i'm not sure hardcoding it to the PV is wrong, but was there a reason you did it this way instead?

Reply:
1. I was thinking the `-XX:ErrorFile` would be an `optional` argument in the deployment config. With that said, I wanted a way to have the log file off by default.
2. I believe the correct place for logs is `/var/log/jenkins`. Reading from https://wiki.jenkins.io/display/JENKINS/Logging. From the user's perspective, I was wondering how intuitive it would be to place the on the PV for `/var/lib/jenkins`. I started created a new PV for logs... But remembered your comment about this being a getting the developer running.

I am fine if we want this to be a ENV var that is defaulted to /var/lib/jenkins/logs. Let me know the direction here. Thanks!

Reply:
i think on further reflection i'm ok leaving it this way and someone who wants to control the path explicitly can set the -XX:ErrorFile arg themselves via the JENKINS_OVERRIDES (and not set the FATAL_ERROR_LOG env variable).

Reply:
@bparees yes that is the conclusion I came to as well. I will make the changes shortly (this is to address all comments). Thanks!